packages/backend/src/api/endpoints/OrganizationRoleApi.ts

```diff
@@ -0,0 +1,98 @@
+import { joinPaths } from '../../util/path';
+import type { DeletedObject } from '../resources';
+import type { OrganizationRole } from '../resources/OrganizationRole';
+import { AbstractAPI } from './AbstractApi';
+
+const basePath = '/organizations_roles';
+
+type GetOrganizationRoleListParams = {
```

Comment:
🔧 We also support `query` & `order_by` parameters

Reply:
@chanioxaris is order_by a string or can i provide a stricter type ?

Reply:
It's just a string